### PR TITLE
Incremental video

### PR DESCRIPTION
**Pull request by HassanJaveed84**
It took 1 hours 26 minutes to completion, overwriting last 3 days of viewing data.  
video table is exactly the same as non-incremental one. 
However, video_timeline table has 49 fewer rows than the non-incremental version. I expected incremental video_timeline to have more rows as I ran the workflow a few hours after video-timeline job had started. Currently investigating the cause. 

Some paths/table names have been changed for testing only.

### CODE REVIEW

File: edx/analytics/tasks/insights/video.py [hunk 1]
```diff
@@ -643,7 +745,7 @@ class VideoUsageTableTask(VideoTableDownstreamMixin, HiveTableTask):
 
     @property
     def table(self):
-        return 'video_usage'
+        return 'video_usage_test'
```
Comment (brianhw) [inline edx/analytics/tasks/insights/video.py]: I assume this is to facilitate testing.  Alternative is to run with different warehouse_path in config.

**Review by brianhw — COMMENTED** [review]
Cursory review finds no offhand problems -- looks good.

**Comment by HassanJaveed84** [comment]
Ready for a final review.

File: edx/analytics/tasks/insights/video.py [hunk 2]
```diff
@@ -505,21 +606,20 @@ class VideoUsageTask(VideoTableDownstreamMixin, MapReduceJobTask):
...
+    def requires_local(self):
+        return UserVideoViewingByDateTask(
             mapreduce_engine=self.mapreduce_engine,
             n_reduce_tasks=self.n_reduce_tasks,
             source=self.source,
             interval=self.interval,
             pattern=self.pattern,
-            output_root=input_path,
+            overwrite_n_days=self.overwrite_n_days,
+            overwrite=True,
```
Comment (brianhw) [inline edx/analytics/tasks/insights/video.py:617]: Just curious if you think this will do the right thing when overwrite_n_days = 0 ...

**Review by brianhw — APPROVED** [review]
One small question, but non-blocking.  Looks good! 👍

**Pull request merged**